pkg/config/derived-service-cert.go

```diff
@@ -48,6 +49,7 @@ func (idCfg *IdentityConfig) derivedServiceCertConfig() error {
 			Provider:          "",
 			AthenzDomainName:  "",
 			AthenzServiceName: "",
+			CertExtraSANDNSs:  []string{},
```

Comment:
I think you should change in a way so that you can set the following in `pkg/certificate/identity.go`:
```go
SANs: util.SubjectAlternateNames{
	DNSNames: idCfg.ServiceCert.CopperArgos.Sans,
	URIs:     []url.URL{*spiffeURI},
},
```
introducing `CertExtraSANDNSs` under `CopperArgosMode` doesn't look clean to me.

Reply:
Made a sample PR: https://github.com/AthenZ/k8s-athenz-sia/pull/179

Reply:
@mlajkim To simplify the PR, could you make the PR against [ctyano:tatyano](https://github.com/ctyano/k8s-athenz-sia/tree/tatyano), so that we can include the both of this original PR and your proposal?

Reply:
done here https://github.com/ctyano/k8s-athenz-sia/pull/11/files by @y-myajima